Algorithms.Tests/Sequences/A000040_PrimesSequenceTests.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Linq;
+using System.Numerics;
+using Algorithms.Sequences;
+using NUnit.Framework;
+
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename class 'A000040_PrimesSequenceTests' to match pascal case naming rules, consider using 'A000040PrimesSequenceTests'.](https://app.codacy.com/app/siriak/C-Sharp/pullRequest?prid=4650688)